pkg/admission/server_test.go

```diff
@@ -59,6 +59,10 @@ func TestServeHTTPInvalidMethod(t *testing.T) {
 }
 
 func TestServeHTTPSubmissions(t *testing.T) {
+
+	const invalidHostname = "!@!.foo.com"
+	const invalidHostnameErrorMsg = "spec.listeners[0].hostname: Invalid value: \"!@!.foo.com\": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')"
```

Comment:
It might have been better to only extract `a lowercase ...` because the part before it has values that can change.

Reply:
ahh I think I follow you know, sorry, let me update with that.